### ISSUE TITLE
Question:  is formvalidation.io compatible with Bootstrap 4 Alpha 6?

### ISSUE THREAD

**Issue by gouldtv** [formvalidation/support#856]
Question:  is formvalidation.io compatible with Bootstrap 4 Alpha 6? I know most of the examples are for Bootstrap 4 alpha 3.  I've got it working, except that I'm getting a "Error: The class FormValidation.Framework.Bootstrap is not implemented" error, yet I have both the bootstrap.min.js linked as well as the bootstrap4.min.js that comes with the distro.

Since it seems to be working, I'm just not sure if I should be worried or not.

**Comment by jipexu** [comment]
yes is compatible with Bootstrap 4 Alpha 6 even with BS beta 2 (we used and works fine)... perhaps a few css hack to keep some old classes ...

**Comment by brettalton** [comment]
how do we get to use it? I only see the old version 0.6.2 which doesn't support bootstrap 4.
